docs/tutorials/new-feature-tutorial.md

```diff
@@ -507,66 +508,78 @@ function _setup_page() {
+    section. New features don't usually lie in this section as it mostly contains
+    the bio of the realm.
 
-* type
-* checked_msg (what message the user sees when they enable the setting)
-* unchecked_msg (what message the user sees when they disable the setting)
+- `org_settings`: This contains properties which belong to organization
+settings section. Settings belonging to this section generally decide what
+features should be available to a user like deleting a message, message
+edit history etc. and usually they affect the whole realm including admin
+(who definitely have access to change them). Our `mandatory_topics`
+feature is going to lie in this section.
+
+- `org_permissions`: This contains properties which belong to organization
+permissions section. These properties are related to the user access to
+the various feature like `bot_creation_policy` which decide who can create bots.
+
+After deciding the section of setting, you've to find the appropriate subsection of the new property
+i.e. under what header your setting will fall into. For example in this case of
+`mandatory_topics` it will lie in "Message feed" (`msg_feed`) subsection.
+
+*If you're not sure in which section your feature will lie it is better
+to discuss it in the [community](https://chat.zulip.org/) before implementing it.*
+
+And the property you're adding, you have to also provide `type` i.e. whether it is
+`bool`, `integer` or `text`.
 
 ``` diff
 
 // static/js/settings_org.js
-
-var property_types = {
-    settings: {
+var org_settings = {
+    msg_editing: {
         // ...
     },
-    permissions: { // ...
+    msg_feed: {
+        // ...
 +       mandatory_topics: {
 +           type: 'bool',
-+           checked_msg: i18n.t("Topics are required in messages to streams"),
-+           unchecked_msg: i18n.t("Topics are not required in messages to streams"),
-        },
++       },
     },
 };
-```
 
-Additionally, any code needed to update the UI when the setting is changed
-should be written in a function inside `settings_org.js`.
-For example, when a realm description is updated, that value change should
-occur in other windows where the description field is visible:
-
-    # static/js/settings_org.js
-
-    exports.update_realm_description = function () {
-        if (!meta.loaded) {
-            return;
-        }
-
-        $('#id_realm_description').val(page_params.realm_description);
-    };
+```
 
+Note that there can be settings like `realm_create_stream_permission` which
+needs some special treatment when they don't match the common pattern like
+most checkboxes. We can't extract the property name and compare the value
+of such input elements with those in `page_params`, so we have to manually
+handle such situations in some functions:
 
-This ensures the appropriate code will run even if the
-changes are made in another browser window.
+- `settings_org.property_value_element_refers`: It processes property name
```

Comment:
We should probably rename this function to something that reads more clearly.  Maybe "settings_org.get_current_property_value" or something?